desktop/cmp/grid/GridModel.js

```diff
@@ -59,6 +63,8 @@ export class GridModel {
      * @param {string} [groupBy] - Column ID by which to group.
      * @param {boolean} [enableColChooser] - true to setup support for column chooser UI and
      *      install a default context menu item to launch the chooser.
+     * @param {boolean} [enableExport] - false to disable export context menu items.
+     * @param {function|string} [exportFilename] - Filename for exported file, or a closure to generate one.
```

Comment:
This closure should probably be passed the gridModel as its first argument for good measure.